tasks/main.yml

```diff
@@ -79,6 +120,22 @@
     mode: u=rw,g=r
   register: apache_conf
 
+- name: "Apache security: ensure 'ServerTokens' to be 'Prod'"
```

Comment:
I'm inclined to drop this commit introducing apache hardening, we should do this in other role since this should be only about making koha work. This would be something that not only applies to Koha and might be useful for other applications running on the server.

Reply:
removed but stored, I will push this hardening code in another role on which we will decide.